valhalla/baldr/graphconstants.h

```diff
@@ -629,6 +629,7 @@ constexpr uint8_t kFreeFlowMask = 1;
 constexpr uint8_t kConstrainedFlowMask = 2;
 constexpr uint8_t kPredictedFlowMask = 4;
 constexpr uint8_t kCurrentFlowMask = 8;
+constexpr uint8_t kNotCurrentFlowMask = kFreeFlowMask | kConstrainedFlowMask | kPredictedFlowMask;
```

Comment:
i dont th ink we need this we can just use bitwise operators directly right?

Reply:
I don't mind. I didn't measure but I hope it won't impact performance

Reply:
bitwise negation of a constant? if the compiler doesnt optimize that then ill be a monkeys uncle!

Reply:
Oh, damn, indeed.. I forgot that they are constants(constexpr-s). The complier definitely handles such things 👍  Thanks for the catch, @kevinkreiser